test/tests/performance/pbs_sched_perf.py

```diff
@@ -271,8 +271,7 @@ def test_many_jobs_with_calendaring(self):
             for i in range(chunk_size):
                 nchunks = i + 1
```

Comment:
Instead of this, you could do for i in range (1, chunk_size + 1) and then you don't need the nchunks variable